src/messenger.py

```diff
@@ -30,19 +30,24 @@ def send_stash(self):
     def send_gold(self):
         self._message_api.send_gold()
 
-    def send_message(self, msg: str):
-        self._message_api.send_message(msg)
+    def send_status(self, msg: str):
+        self._message_api.send_status(msg)        
+
+    def send_message(self, title: str, msg: str, img: str):
+        self._message_api.send_message(title, msg, img)
```

Comment:
This call will throw an error for the generic API because it does not have img as an argument. You will have to add img as an optional argument e.g. with default = None

Reply:
You will have to add the "img" param to the send_message() function in generic_api.py